draft-irtf-cfrg-opaque.md

```diff
@@ -1406,68 +1402,62 @@ following fields:
 {{ake-client}} and {{ake-server}} specify the inner workings of client and
 server functions, respectively.
 
-### AKE Group {#key-creation}
+### 3DH Primitives {#key-creation}
 
-We assume the following functions to exist for all candidate groups in this
+We assume the following functions to exist for all Diffie Hellman groups in this
 setting:
 
-- DeriveAuthKeyPair(seed): Derive a private and public authentication key pair
-  deterministically from the input `seed`.
-- GenerateAuthKeyPair(): Return a randomly generated private and public key
-  pair. This can be implemented by invoking DeriveAuthKeyPair with `Nseed`
-  random bytes as input.
-- SerializeElement(element): A function that maps `element` to a unique byte
-  array, mirrored from the definition of the similarly-named function of the
-  OPRF group described in {{OPRF, Section 2.1}}.
-- DiffieHellman(k, B): A function that performs the Diffie-Hellman operation
-  between the scalar input `k` and element `B`.
+- DeriveDHKeypair(seed): Derive a private and public DiffieHellman
+  key pair deterministically from the input `seed`. The type of the private key
+  depends on the implementation, whereas the type of the public key is a
+  byte string of `Npk` bytes.
+- DH(k, B): A function that performs the Diffie-Hellman operation
```

Comment:
Why change the name here from "DiffieHellman(k, B)" to "DH(k, B)"? The latter feels less descriptive, and I am in favor of keeping it as DiffieHellman unless it conflicts with something else...

Reply:
Reverted.